lib/matplotlib/rcsetup.py

```diff
@@ -1150,7 +1150,8 @@ def _validate_linestyle(ls):
     'image.interpolation': ['nearest', validate_string],
     'image.cmap':          ['viridis', validate_string],        # one of gray, jet, etc
     'image.lut':           [256, validate_int],  # lookup table
-    'image.origin':        ['upper', validate_string],  # lookup table
+    'image.origin':        ['upper',
+                            ValidateInStrings('image.origin', ['upper', 'lower'])],
```

Comment:
Surprised you didn't wrap this in something that wasn't camel case :wink:

Reply:
I have bigger plans for rcsetup, but they're shelved for now...